ui/src/reducers/backoffice.js

```diff
@@ -84,6 +106,20 @@ const BackofficeReducer = (state = initialState, action) => {
         .set('loading', false);
     case BACKOFFICE_DELETE_ERROR:
       return state.set('loading', false);
+    case BACKOFFICE_DASHBOARD_REQUEST:
+      return state.set('dashboard', { loading: true });
+    case BACKOFFICE_DASHBOARD_SUCCESS: {
```

Comment:
we can actually simplify that part. As we need these info only for displaying the tabs we can be specific and already separate it them here in literature and authors. This will also simplify the component logic and will make debugging easier.